Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -121,13 +121,12 @@ TEST_F(SerializerTest, EncodesBoolProtoToBytes) {
     std::vector<uint8_t> bytes;
   };
 
-  // TEXT_FORMAT_PROTO: 'boolean_value: true'
-  std::vector<uint8_t> true_bytes{0x08, 0x01};
-  // TEXT_FORMAT_PROTO: 'boolean_value: false'
-  std::vector<uint8_t> false_bytes{0x08, 0x00};
+  std::vector<TestCase> cases{// TEXT_FORMAT_PROTO: 'boolean_value: true'
```

Comment:
lolwut. Does it end up any different if you do `cases = {` ?

Reply:
No; it just indents the other lines by a further 2 spaces. :(